x-pack/plugins/data_usage/public/hooks/use_get_usage_metrics.ts

```diff
@@ -31,8 +31,9 @@ export const useGetDataUsageMetrics = (
     queryKey: ['get-data-usage-metrics', body],
     ...options,
     keepPreviousData: true,
-    queryFn: async () => {
+    queryFn: async ({ signal }) => {
       return http.post<UsageMetricsResponseSchemaBody>(DATA_USAGE_METRICS_API_ROUTE, {
+        signal,
```

Comment:
wow, that was easy 😄

Reply:
Yeah, one of many reasons a lot of us moved away from redux to manage server state and use tanstack-query.